WordPress/src/main/java/org/wordpress/android/ui/reader/services/discover/ReaderDiscoverLogic.kt

```diff
@@ -176,6 +183,12 @@ class ReaderDiscoverLogic(
         ReaderPostTable.addOrUpdatePosts(ReaderTag.createDiscoverPostCardsTag(), postList)
     }
 
+    private fun insertBlogsIntoDb(blogs: List<ReaderBlog>) {
+        blogs.forEach { blog ->
+            ReaderBlogTable.addOrUpdateBlog(blog)
+        }
```

Comment:
I've decided to reuse `ReaderBlogTable .addOrUpdateBlog()` function because we don't store a lot of blogs at this point, but I can add a new method that will store all the blogs in one transaction. WDYT?

Reply:
I think the current approach is fine ;)